dev-client/src/components/inputs/Select.tsx

```diff
@@ -152,11 +100,43 @@ export const Select = <T, Nullable extends boolean>({
           </Row>
         </Pressable>
       )}>
-      <BottomSheetFlatList
-        ListHeaderComponent={Header}
-        data={optionsWithNull}
-        renderItem={renderOption}
-      />
+      <BottomSheetScrollView>
+        <Pressable
+          onPress={onClose}
+          accessibilityLabel={t('general.done')}
+          accessibilityRole="button">
+          <Row
+            backgroundColor="primary.main"
+            justifyContent="flex-end"
+            alignItems="center"
+            padding="md">
+            <Text
+              variant="body1-strong"
+              color="primary.contrast"
+              textTransform="uppercase">
+              {t('general.done')}
+            </Text>
+          </Row>
+        </Pressable>
+        <MenuList>
+          {optionsWithNull.map((option, i) => {
+            const itemLabel = option ? renderValue(option) : unselectedLabel!;
+            const selected = option === value;
+            return (
+              <MenuItem
+                key={itemLabel + i}
```

Comment:
Would appreciate a double-check here that this is ok; the item lists tend not to change often + the component is simple so the cost of re-rendering if the list gets reordered etc seemed low, vs. requiring unique IDs as an input to `Select` which seemed out-of-scope for this ticket.

Reply:
This seems fine to me, at least for the way we use it. May be worth a comment just in case in the future we do end up with a `renderValue` that's expensive in conjunction with a reorder-able list -- but I agree that seems out-of-scope for now.

Reply:
Mostly just thinking aloud here -- I guess another potential issue is reordering in conjunction with multiple items with the same `renderValue`s, or an item whose `renderValue` is the same as the unselectedLabel... but that also seems rather unlikely

Reply:
After thinking about it, you raise a good point here. I'll see if I can just require that client code provides unique keys for select items.